client/src/components/ProfileDetailsCard/ProfileDetailsCard.tsx

```diff
@@ -7,7 +7,16 @@ interface PropTypes {
   profile: Profile;
 }
 
+const placeholderCoverPhoto = 'https://www.kimballstock.com/images/dog-stock-photos.jpg';
+const placeholderGallery = [
+  'https://i.picsum.photos/id/1062/5092/3395.jpg?hmac=o9m7qeU51uOLfXvepXcTrk2ZPiSBJEkiiOp-Qvxja-k',
+  'https://i.picsum.photos/id/1025/4951/3301.jpg?hmac=_aGh5AtoOChip_iaMo8ZvvytfEojcgqbCH7dzaz-H8Y',
+];
+
 const ProfileDetailsCard = ({ profile }: PropTypes) => {
+  profile.coverPhoto = placeholderCoverPhoto;
+  profile.uploadedImages = placeholderGallery;
+
```

Comment:
I feel as though this ticket should conclude any remaining integration - instead of using placeholders like these, we should use the values we fetch from the backend.

I'd highly recommend looking into creating a basic seeding script so we can have some users/profiles in the db (you can use the mock data you have so far as a starting point).